datahub-graphql-core/src/main/resources/module.graphql

```diff
@@ -133,6 +133,15 @@ input AssetCollectionModuleParamsInput {
   The list of asset urns for the asset collection module
   """
   assetUrns: [String!]!
```

Comment:
we could make this optional now that it could be not relevant. but i guess we could always return an empty list as i imagine we do now with your change if there are none